### DISCUSSION TITLE
How to get rid of the extensions?

### DISCUSSION THREAD

**Discussion by amadeus3301** [arichornlover/uYouEnhanced#726] [Q&A]
While installing i chose the "keep extension" option, now i wish to have them removed to free up app id slots

I understand i have to wait and let them expire, but can i refresh the main app itself, without it having refresh the extensions as well? Or do i have to let the whole app itself expire and THEN reinstall?

However, i did remove the app once and tried to reinstall it, but it did not give me the prompt for installing the extensions as well. 

TLDR - please guide me as to how i remove the extensions

**Comment by arichornlover** [comment]
> I understand i have to wait and let them expire, but can i refresh the main app itself, without it having refresh the extensions as well? Or do i have to let the whole app itself expire and THEN reinstall?

You can reinstall the ipa and choose no extensions but unfortunately you gotta wait for those App IDs to expire. and here’s the AltStore FAQ about it here if you want to check it out. https://faq.altstore.io/how-to-use-altstore/app-ids

**Reply by amadeus3301** [reply]
I let the youtube extensions expire, reinstalled the app, it didn't ask me for a prompt, it just installed the extensions alongside the app and after an entire week i'm back at square one 

Edit: i installed from a repo, maybe that's why, instead of an isolated ipa
